### ISSUE TITLE
Cypher $auth.jwt.sub in cypher does not work anymore

### ISSUE THREAD

**Issue by eriklueth** [neo4j/graphql#3772]
**Describe the bug**
In the new beta it looks like you cant use $auth.jwt.sub inside the cypher directive anymore. It seems like you now have to use $jwt.sub now instead and the auth is object is gone.

```
    # This works
    testCypherNoAuth: String! @cypher(statement: "Match (this) return $jwt.sub as result", columnName: "result")
    # This not
    testCypherAuth: String! @cypher(statement: "Match (this) return $auth.jwt.sub as result", columnName: "result")
```
**Expected behavior**
Keep old auth object or update the docs.

**Comment by neo4j-team-graphql** [comment]
Many thanks for raising this bug report @eriklueth. :bug: We will now attempt to reproduce the bug based on the steps you have provided.

Please ensure that you've provided the necessary information for a minimal reproduction, including but not limited to:
* Type definitions
* Resolvers
* Query and/or Mutation (or multiple) needed to reproduce

If you have a support agreement with Neo4j, please link this GitHub issue to a new or existing Zendesk ticket.

Thanks again! :pray:

**Comment by eriklueth** [comment]
It also seems like auth is not global anymore. So when I use the OGM to create something and the selection set has a cypher with $jwt.sub in it, I get   @neo4j/graphql:execute Neo4jError: Expected parameter(s): jwt because the executed call to neo4j does not contain the jwt in the params.

**Comment by darrellwarde** [comment]
Seems the docs just need updating on this one - it should be `$jwt.sub` moving forwards.

The OGM not working is another issue!

**Comment by darrellwarde** [comment]
I've addressed these issues in the 4.0.0 docs here: https://github.com/neo4j/docs-graphql/pull/17

They will be merged and published soon. 🙂
